Formula/p/pegtl.rb

```diff
@@ -15,13 +16,18 @@ class Pegtl < Formula
   fails_with gcc: "5"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args,
-                            "-DPEGTL_BUILD_TESTS=OFF",
-                            "-DPEGTL_BUILD_EXAMPLES=OFF",
-                            "-DCMAKE_CXX_STANDARD=17"
-      system "make", "install"
-    end
+    odie "update license to BSL-1.0" if version > "3.2.7"
+
```

Comment:
```suggestion
```

Similarly, I don't see any reason for this. Just let the audit automatically fail by not adding to exception list.